### ISSUE TITLE
Can't reproduce the AP in the paper

### ISSUE THREAD

**Issue by Artificial-Inability** [UX-Decoder/DINOv#11]
Hi, I tried to use coco panoptic_train2017.json to train the DINOv SwinT model, I'm using 64 V100 GPU(total bs=64) to train the model for 36 epochs. The learning rate is 1e-4 at first and drop to 1e-5 and 1e-6 in epoch 24 and epoch 30 respectively. Then I evaluate the model on panoptic_val2017.json. The PQ/mask AP/box AP is 43.3/35.2/37.2, which is lower than the Table 6 in paper (48.9/41.7/45.9). 

I'm using the same config in this repo with minimal changes, can you check my learning rate and schedule and tell me if there's any difference from your ablation experiments in Table6?

I also noticed that the AP of DINOv SwinT trained with COCO+SA1B in Table 1 (49.0/41.5/45.2) is different from Table 6 (49.6/42.7/47.0), is that from two different experiments with the same setting?

**Comment by FengLi-ust** [comment]
Thanks for your comment. We train for 50 epochs. I will also check the code. BTW, have you reproduced the results using our checkpoint?

**Comment by Artificial-Inability** [comment]
Thanks for your information, I evaluated the DINOv checkpoint you provided on panoptic_val2017.json and got reasonable results (45.7/42.0/50.1 for swinT and 54.2/50.4/57.7 for swinL). After I solved the issue in https://github.com/UX-Decoder/DINOv/issues/5#issuecomment-2008714195 I also get similar results using instances_val2017.json so I think there's no problem in your checkpoint and evaluation part.

I still want to align the details of training schedule, in which epoch do you drop the lr from 1e-4 to 1e-5 (and 1e-6 if 3step) ?

**Comment by FengLi-ust** [comment]
Drop at around 40 epochs

**Comment by Artificial-Inability** [comment]
@FengLi-ust Hi, I have finished the 50epoch dinov_swint COCO Panoptic training. When I use lr=1e-4 the PQ/Mask AP/Box AP is 45.5/37.3/40.1 and when I use lr=2e-4 the PQ/Mask AP/Box AP is 47.4/39.3/43.1. Though it should be a bit higher in my expectation, I think its a reasonable result now. Let me know if you are using a different lr or other different settings when you train COCO without SA1B, Thanks!

Now I'm considering training with both COCO and SA1B. You are taking exactly one COCO image and one SA1B image for each iteration, so I assume the coco images will repeat more times than SA1B images. Then, how to set the SOLVER.MAX_ITER in this case? Let's assume we are using ~120k COCO images and 2000k SA1B images and train 50ep with 64GPU, should I set it to 2000000 * 50 / 64 = 1562500?

**Comment by zzz123123123123** [comment]
> 感谢您的信息，我评估了您在 panoptic_val2017.json 上提供的 DINOv 检查点并获得了合理的结果（swinT 的 PQ/掩码 AP/盒 AP 为 50.1/42.0/45.7，swinL 为 57.7/50.4/54.2）。在我解决了 [#5（评论）](https://github.com/UX-Decoder/DINOv/issues/5#issuecomment-2008714195)中的问题后，我使用 instances_val2017.json 也得到了类似的结果，所以我认为你的检查点和评估部分没有问题。
> 
> 我仍然想调整训练计划的细节，在哪个纪元中将 lr 从 1e-4 降至 1e-5（如果是 3step，则为 1e-6）？

您好，想请教一下，您在panoptic_val2017.json上获得了与论文一致的结果吗（swinL 为 57.7/50.4/54.2），为什么我一直比论文低一个百分点。请问一下您是先生成的id_139_idx_1.safetensors这种文件在评估的吗，抱歉打扰你了
